Tests/Mock/FosUser/MailerMock.php

```diff
@@ -1,5 +1,14 @@
 <?php
 
+/*
+ * This file is part of the DunglasApiBundle package.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
```

Comment:
If you grant me the access to StyleCI yes ;)

Otherwise just log in, click on DunglasApiBundle -> settings and check the "Header Comment" section to paste the following text:

```
This file is part of the DunglasApiBundle package.

(c) Kévin Dunglas <dunglas@gmail.com>

For the full copyright and license information, please view the LICENSE
file that was distributed with this source code.
```